data.yml

```diff
@@ -2054,6 +2088,144 @@ components:
+        paymentDueDate: 
+          type: string
+          description: >- 
+            (mandatory) - Date that the next payment for the card is due. 
+    loanaccountMetadata: 
+      type: object
+      properties: 
+        startDate: 
+          type: string
+          description: >- 
+            (optional) - start date for the loan.
+        endDate:
+          type: string
+          description: >-
+            (optional) - Date that the loan is due to be repaid in full.
+        repaymentType:
+          type: string
+          description: >-
+            (optional) - Options in place for repayments. If absent defaults to PRINCIPAL_AND_INTEREST.
+          enum:
+          - INTEREST_ONLY
+          - PRINCIPAL_AND_INTEREST
+        minInstalmentAmount:
+          type: string
+          description: >- 
+            (optional) - Minimum amount of next instalment. 
+    lendingRateaccountMetadata:
+      required:
+      - lendingRateType
+      - rate
+      type: object
+      properties: 
+        lendingRateType:
+          type: string
+          description: >-
+            (mandatory) - The type of rate (fixed, variable, etc).
+          enum: 
+          - BUNDLE_DISCOUNT_FIXED
+          - BUNDLE_DISCOUNT_VARIABLE
+          - CASH_ADVANCE
+          - DISCOUNT
+          - FIXED
+          - FLOATING
+          - INTRODUCTORY
+          - MARKET_LINKED
+          - PENALTY
+          - PURCHASE
+          - VARIABLE
+          - IN_ADVANCE
+          - IN_ARREARS
+          - INTEREST_ONLY
+          - PRINCIPAL_AND_INTEREST
+          - INVESTMENT
+          - OWNER_OCCUPIED
+        rate: 
+          type: string
+          description: >- 
+            (mandatory) - The rate to be applied. 
+        applicationFrequency: 
+          type: string
+          description: >- 
+            (optional) - The period after which the calculated amount(s) are applied (i.e. debited or credited) to the account.
+    depositRateaccountMetadata: 
+      required:
+      - depositRateType
+      - rate
+      type: object
+      properties: 
+        depositRateType:
+          type: string
+          description: >-
+            (mandatory) - The type of rate (base, bonus, etc).
+          enum:
+          - BONUS
+          - BUNDLE_BONUS
+          - FIXED
+          - FLOATING
+          - INTRODUCTORY
+          - MARKET_LINKED
+          - VARIABLE
+        rate: 
+          type: string
+          description: >- 
+            (mandatory) - The rate to be applied. 
+        applicationFrequency: 
+          type: string
+          description: >- 
+            (optional) - The period after which the calculated amount(s) are 'applied' (i.e. debited or credited) to the account.
+    feesaccountMetadata: 
+      required:
+      - name
+      - feeType
+      type: object
+      properties:
+        name:
+          type: string
+          description: (mandatory Field) Name of the fee
```

Comment:
Inconsistent use of (mandatory) and (mandatory Field). Also some have `-` and some don't.